src/cscript.c

```diff
@@ -179,7 +179,7 @@ static tree __evaluate_cscript(tree head, const char *fname, int argc,
     if (has_main)
         add_call_to_main(head, num_main_args);
 
-    return evaluate(head, fname);
+    evaluate(head, fname);
```

Comment:
Control may reach the end of a non-`void` function here. The signature is:
```c
static tree __evaluate_cscript()
```

..but it returns nothing.